backend/climateconnect_api/tasks.py

```diff
@@ -1,9 +1,50 @@
+import logging
+from datetime import timedelta
+from typing import List
+
 from climateconnect_main.celery import app
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+from climateconnect_api.models import UserNotification
+from climateconnect_api.utility.email_setup import \
+    send_email_reminder_for_unread_notifications
 
-import logging
 logger = logging.getLogger(__name__)
 
 
 @app.task
-def add(a, b):
-    logger.info(f"testing.... {a+b}")
+def schedule_automated_reminder_for_user_notifications():
+    # Get all user_ids for people who have not checked their notification
+    all_user_ids = list(
+        UserNotification.objects.filter(
+            read_at__isnull=True,
+            created_at__lte=(timezone.now() - timedelta(days=2))
```

Comment:
I'd only do this for the private message notifications. Doing this for project likes etc. will not be necessary imo.

Reply:
Not even group messages?

Reply:
In the first step, I'd only do it for private messages because those are for sure addressed to you while group messages don't need to be addressed to you. If we also do it for group messages we have to make sure that we still respect their settings, if they disabled notifications on group messages. In this case we could maybe say You missed messages in group x and you missed private messages.

Reply:
Done